01_P/P_2_1_2_03/sketch.js

```diff
@@ -18,36 +18,34 @@
 
 /**
  * changing size of circles in a rad grid depending the mouseposition
- *   
+ *
  * MOUSE
  * position x/y        : module size and offset z
- * 
+ *
  * KEYS
  * s                   : save png
  */
- 
+'use strict';
 
 var tileCount = 20;
 var moduleColor;
 var moduleAlpha = 180;
 var actRandomSeed = 0;
-var max_distance = 500; 
-
+var max_distance = 500;
 
 function setup(){
   createCanvas(600, 600);
+  noFill();
+  strokeWeight(3);
   moduleColor = color(0, 0, 0, moduleAlpha);
 }
 
 function draw() {
   background(255);
-  smooth();
```

Comment:
Removed `smooth()` since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).